avm/res/web/site/main.bicep

```
@@ -285,6 +291,25 @@ module app_authsettingsv2 'config--authsettingsv2/main.bicep' = if (!empty(authS
   }
 }
 
+module app_logssettings 'config--logs/main.bicep' = if (!empty(logsConfiguration)) {
+  name: '${uniqueString(deployment().name, location)}-Site-Config-Logs'
+  params: {
+    appName: app.name
+    logsConfiguration: logsConfiguration ?? {}
```

```suggestion
    logsConfiguration: logsConfiguration
```

The deployment's condition should render this redundant